src/actions/sources/newSources.js

```diff
@@ -81,12 +81,16 @@ function loadSourceMap(sourceId: SourceId) {
     }
 
     if (!urls) {
+      // The source might have changed while we looked up the URLs, so we
```

Comment:
Wow, nice find.  I guess my only suggestion would be being more specific in the comment, as to what could have possibly changed with the source.  Loading status?